app/graphql/mutations/import_gtfs_feeds.rb

```diff
@@ -0,0 +1,27 @@
+module Mutations
+  class ImportGtfsFeeds < BaseMutation
+    argument :dataprovider_id, ID, required: false
+    argument :feed, String, required: false
+
+    type Types::StatusType
+
+    def resolve(dataprovider_id:, feed:)
+      status, status_code = ["Import failed: No data provider found", 404] unless dataprovider_id.present?
+
+      if dataprovider_id.present?
+        # TODO: Implement feed parameter to start only given feed and not all feeds
+        #   - check if data_provider exists
+        #   - filter for feed parameter
+        GtfsImporterJob.perform_later
+        status, status_code = ["Import started", 200]
+      end
+
+      OpenStruct.new(
+        id: dataprovider_id,
```

Comment:
why is the data provider id being returned just as `id`?

```
{
  "data": {
    "importGtfsFeeds": {
      "id": "64",
      "status": "Import started",
      "statusCode": 200
    }
  }
}
```

this looks like the id is an id from a gtfs feed that gets imported.
i suggest returning `data_provider_id` instead of `id`

Reply:
we want to keep the structure with id and return nil in this case